releases/release-1.10/release-notes-draft.md

```diff
@@ -1,3 +1,1018 @@
-# Release 1.10
+# Kubernetes 1.10 Release Notes Draft
 
-This will will be used to track release notes for Kubernetes 1.10.
+[[TOC]]
```

Comment:
please drop as will be added automatically

Reply:
Done.